app/src/main/java/com/gh4a/adapter/timeline/CommentViewHolder.java

```diff
@@ -0,0 +1,197 @@
+        ivMenu.setOnClickListener(this);
+
+        mPopupMenu = new PopupMenu(view.getContext(), ivMenu);
+        mPopupMenu.getMenuInflater().inflate(R.menu.comment_menu, mPopupMenu.getMenu());
+        mPopupMenu.setOnMenuItemClickListener(this);
+
+        MenuItem reactItem = mPopupMenu.getMenu().findItem(R.id.react);
+        mPopupMenu.getMenuInflater().inflate(R.menu.reaction_menu, reactItem.getSubMenu());
+
+        mReactionMenuHelper = new ReactionBar.AddReactionMenuHelper(view.getContext(),
+                reactItem.getSubMenu(), this, this, reactionDetailsCache);
+    }
+
+    @Override
+    public void bind(TimelineItem.TimelineComment item) {
+        mBoundItem = item;
+
+        User user = item.comment.getUser();
+        Date createdAt = item.comment.getCreatedAt();
+        Date updatedAt = item.comment.getUpdatedAt();
+
+        AvatarHandler.assignAvatar(ivGravatar, user);
+        ivGravatar.setTag(user);
+
+        tvTimestamp.setText(StringUtils.formatRelativeTime(mContext, createdAt, true));
+        if (createdAt.equals(updatedAt) || item.getCommitComment() != null) {
+            // Edit timestamp looks to be incorrect for commit comments. They all appear to be
+            // modified.
+            tvEditTimestamp.setVisibility(View.GONE);
+        } else {
+            tvEditTimestamp.setText(StringUtils.formatRelativeTime(mContext, updatedAt, true));
+            tvEditTimestamp.setVisibility(View.VISIBLE);
+        }
+
+        // Body
+        mImageGetter.bind(tvDesc, item.comment.getBodyHtml(), item.comment.getId());
+
+        // Extra view
+        String login = ApiHelpers.getUserLogin(mContext, item.comment.getUser());
+        SpannableString userName = new SpannableString(login);
+        userName.setSpan(new StyleSpan(Typeface.BOLD), 0, userName.length(), 0);
+        tvExtra.setText(userName);
+
+        if (mCallback.canQuote()) {
+            tvDesc.setCustomSelectionActionModeCallback(
+                    new UiUtils.QuoteActionModeCallback(tvDesc) {
+                @Override
+                public void onTextQuoted(CharSequence text) {
+                    mCallback.quoteText(text);
+                }
+            });
+        } else {
+            tvDesc.setCustomSelectionActionModeCallback(null);
+        }
+
+        ivMenu.setTag(item);
+
+        // Reactions
+        reactions.setReactions(item.comment.getReactions());
+
+        String ourLogin = Gh4Application.get().getAuthLogin();
+        boolean canEdit = ApiHelpers.loginEquals(user, ourLogin) ||
+                ApiHelpers.loginEquals(mRepoOwner, ourLogin);
+
+        Menu menu = mPopupMenu.getMenu();
+        menu.findItem(R.id.edit).setVisible(canEdit);
+        menu.findItem(R.id.delete).setVisible(canEdit);
+        menu.findItem(R.id.view_in_file).setVisible(
+                item.file != null && item.getCommitComment() != null &&
+                        item.getCommitComment().getPosition() != -1);
```

Comment:
I didn't (consciously) follow a convention there, but I think I usually place && or || at the start of the line. I *do* try to avoid double indention of a single statement whereever possible though.